hs/t/n/Err_Alias_Name_Clash.rsh

```diff
@@ -0,0 +1,16 @@
+'reach 0.1';
+
+export const main = Reach.App(() => {
+  const A = View('A', {
+    x: UInt
+  }, {
+    x: ['y', 'z']
+  });
+  const B = View('B', {
+    xp: UInt
+  }, {
+    xp: ['y']
```

Comment:
The documentation had better make clear that the interface is *not* included automatically with aliases

Reply:
This is now inconsistent with your doc change. This file should define the following functions: `A_x, A_y, A_z, B_xp, B_y`. There is no name clash there. This suggests the documentation is wrong or the implementation. I think the implementation is probably the one at fault.